episodes/enabling-technologies-standards.Rmd

```diff
@@ -0,0 +1,699 @@
+A simple JSON schema could look like the one below. It declares:
+<ul>
+  <li>JSON Schema version with `$schema`</li>
+  <li>a list (an array) of required (i. e. mandatory) properties</li>
+  <li>one required property (i.e. ``"superhero"``)</li>
+  <li>one optional property (i.e. ``"power"``)</li>
+  <li>data type constraints for record values (e.g. ``"type": "integer"``)</li>
+</ul>
+    
+There are also some `descriptions` added for the human reader.
+
+```json
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "description": "In real life you would add a meaningful description here.",
+  "type": "object",
+  "required": [
+    "superhero"
+  ],
+  "properties": {
+    "superhero": {
+      "description": "A mandatory string property.",
+      "type": "string"
+    },
+    "power": {
+      "description": "An optional numeric property.",
+      "type": "integer"
+    }
+  }
+}
+
+```
+A JSON instance is syntactically valid, if it conforms to the definition described by the JSON schema. Note, that the JSON Schema `required` keyword holds a list of keys that must be present for a JSON object to be considered a valid instance of this schema.
+
+```json
+{
+  "superhero": "I am just a string"
+}
+```
+
+:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: callout 
+The most challenging part of schema development can be to **have everyone agree on the same expectations.**
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
+## Lecture: Challenge introduction
+
+**Slide set**: <a href="../files/slideSet/ChallengeIntroduction_4_5_CarpentriesIncubator_ScientificMetadata.pdf" target="_blank" download>Challenge4and5.pdf</a>
+
+**Time estimate**: 5 min
+
+1. Let's briefly recap on the most important characteristics of a JSON schema.
+  - each of the properties in the schema are described in objects.
+  - the data type of a property is specified by the key `type`.
+  - if you want to further specify the format of a data entry, use the key `format`.
+  - mandatory data entries are given by the key `required` in an array.
+  - for controlled lists, you specify the given values in an array and the key `enum`.
+2. The complete specifications for JSON schema can be found on json-schema.org.
+3. If you develop a schema, that is supposed to be used by a group of people, the most challenging part can be to have everyone agree on the same expectations.
+3. <i>(Allow for some time for questions.)</i>
+4. We will now head back to our roller coaster data. In your collaboration you spend some time in discussing your expectations on the metadata, that is necessary to record with the data. But finally you have created an example JSON object, basically the ideal metadata record for the experiment within your collaboration. To enforce this metadata structure, you want to write a JSON schema. As this can get very lengthy...
```

Comment:
Is this about a hypothetical "real life situation", or about the actual break-out groups?

Because I was confused by the "collaboration"